src/app.js

```diff
@@ -1,11 +1,28 @@
 const express = require('express');
+const session = require('express-session');
+const passport = require('passport');
```

Comment:
You should actually import `passport` from your custom utils file and not the package. Because that is where you've setup the Google strategy.

Reply:
Makes sense. Thank you for clarifying.